OpenLIFULib/OpenLIFULib/transducer.py

```diff
@@ -64,17 +71,50 @@ def initialize_from_openlifu_transducer(
 
         transform_matrix_vtk = numpy_to_vtk_4x4(transform_matrix_numpy)
         transform_node.SetMatrixTransformToParent(transform_matrix_vtk)
+        shNode.SetItemParent(shNode.GetItemByDataNode(transform_node), parentFolderItem)
+
+        #Model nodes
+        model_node = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLModelNode")
+        model_node.SetName(f"{slicer_transducer_name}-transducer")
+        model_node.SetAndObservePolyData(transducer.get_polydata())
+        model_node.SetAndObserveTransformNodeID(transform_node.GetID())
+        shNode.SetItemParent(shNode.GetItemByDataNode(model_node), parentFolderItem)
         model_node.CreateDefaultDisplayNodes() # toggles the "eyeball" on
 
-        return SlicerOpenLIFUTransducer(
-            SlicerOpenLIFUTransducerWrapper(transducer), model_node, transform_node
+        if transducer.transducer_body_filename:
+            body_model_node = slicer.util.loadModel(Path(parent_dir)/transducer.transducer_body_filename)
+            body_model_node.SetName(f"{slicer_transducer_name}-body")
+            body_model_node.SetAndObserveTransformNodeID(transform_node.GetID())
+            shNode.SetItemParent(shNode.GetItemByDataNode(body_model_node), parentFolderItem)
+        else:
+            body_model_node = None
+
+        if transducer.registration_surface_filename:
+            surface_model_node = slicer.util.loadModel(Path(parent_dir)/transducer.registration_surface_filename)
+            shNode.SetItemParent(shNode.GetItemByDataNode(surface_model_node), parentFolderItem)
+            surface_model_node.SetAndObserveTransformNodeID(transform_node.GetID())
+            surface_model_node.SetName(f"{slicer_transducer_name}-surface")
+        else:
+            surface_model_node = None
```

Comment:
The `Path(parent_dir)/transducer.registration_surface_filename` is making assumptions about how the openlifu database works. Having `initialize_from_openlifu_transducer` accept a `parent_dir` makes sense if we're using it to load from a json file directly, but when we load a transducer via the database I think we should be relying on what `openlifu` provides us in order to load the transducer body and surface models. Can we solve this using a similar method to what you came up with for photoscan data in #158? It may require exposing something new in `openlifu`. It may require exposing something new in `openlifu`